frontend/src/metabase/visualizations/visualizations/CartesianChart/use-models-and-option.ts

```diff
@@ -114,6 +115,8 @@ export function useModelsAndOption({
       return {};
     }
 
+    const shouldAnimate = !isPlaceholder && !isCypressActive;
+
```

Comment:
Any chance we could use the `@prefers-reduced-motion` media setting for this? IIRC it's already disabled in Cypress and I suppose people won't expect charts to be animated when the setting is on as well

Reply:
Probably worth discussing in #43975